src/tmx/TmxApi/CMakeLists.txt

```diff
@@ -43,8 +43,11 @@ IF (NOT TMXAPI_NO_STATIC)
     ENDIF ()
     TARGET_INCLUDE_DIRECTORIES (${PROJECT_NAME}Static SYSTEM PUBLIC
                                 $<BUILD_INTERFACE:${CMAKE_CURRENT_SOURCE_DIR}>
+                                /opt/carma/include/carma_j2735
                                 ${Boost_INCLUDE_DIRS}) 
-    TARGET_LINK_LIBRARIES (${PROJECT_NAME}Static ${ASN_J2735_LIBRARIES} ${Boost_LIBRARIES} ${ICU_LIBRARIES} pugixml)
+    TARGET_LINK_LIBRARIES (${PROJECT_NAME}Static libasn_j2735 ${Boost_LIBRARIES} ${ICU_LIBRARIES} pugixml)
```

Comment:
lets link ::carma_j2735 directly instead of the alias

Reply:
Addressed https://github.com/usdot-fhwa-OPS/V2X-Hub/pull/654/commits/ef1238953d99aa1f7aa13b59a7133c1131039ee5